python/python/bystro/ancestry/preprocess_1kgp_using_gnomad_loadings.sh

```diff
@@ -0,0 +1,37 @@
+#Extract gnomad loadings variant list from 1kgp genomes using plink
+
+#TODO convert this process to one using our own vcf parser
+
+#Download plink2 - can be downloaded with example below or at https://www.cog-genomics.org/plink/2.0/
+#wget https://s3.amazonaws.com/plink2-assets/plink2_linux_avx2_20230707.zip
+#unzip plink2_linux_avx2_20230707.zip 
+
+#Download 1kgp genomes - make sure this is the most recent version
+wget 'ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/data_collections/1000G_2504_high_coverage/working/20220422_3202_phased_SNV_INDEL_SV/1kGP_high_coverage_Illumina.chr'*
+
```

Comment:
let's add a checksum to ensure that we've got the data we think we do

Reply:
Should I add one for each of the files? There's I think 46 that come through at this point. Also to that end, should I add a step to delete them all at the end?

Reply:
Ok I found the checksums in the manifest so was able to easily include from there